### PR TITLE
issue #2 -validation fix and case problem

### PR DESCRIPTION
**Pull request by Kolawole-Akinwande**
this pR fixes issue #2

### CODE REVIEW

File: colorcheck.py [hunk 1]
```diff
@@ -2,22 +2,22 @@
...
+    """
+    In the while loop ask the user to enter a color,
+    lowercase it and store it in a variable. Next check: 
+    - if 'quit' was entered for color, print 'bye' and break. 
+    - if the color is not in VALID_COLORS, print 'Not a valid color' and continue.
+    - otherwise print the color in lower case.
+    """
     while True:
-        intake = input('enter a color' )
-        intake.lower()
```
Comment (Kolawole-Akinwande) [inline colorcheck.py:12]: why did you do this?

File: colorcheck.py [hunk 2]
```diff
@@ -2,22 +2,22 @@
...
+        intake = input('Enter a color: ').lower()
         if intake == "quit":
             print("bye")
             break
-        if intake not in VALID_COLORS:
+        elif intake not in VALID_COLORS:
```
Comment (Kolawole-Akinwande) [inline colorcheck.py:17]: nice one!

File: colorcheck.py (hunk 1 above)
Comment (Kolawole-Akinwande) [reply colorcheck.py:12]: intake.lower() is not assigned to any variable so its useless.

**Review by sdamhub — APPROVED** [review]
Great correction sir

**Pull request merged**